examples/threathunter/threat_intelligence/feed_helper.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime, timedelta, tzinfo
+
+
+class UTC(tzinfo):
+    """UTC"""
+
+    def utcoffset(self, dt):
+        return timedelta(0)
+
+    def tzname(self, dt):
+        return "UTC"
+
+    def dst(self, dt):
+        return timedelta(0)
```

Comment:
Maybe the code needs to force the use of UTC for some reason? Do the builtin methods default to localtime?

Reply:
But it could literally use the same exact methods inside of the datetime lib. These returns have no use, they are simple 14 lines of code that could be done in 0.

For instance on line 7, it is literally using a datetime function (timedelta) to override [this method](https://docs.python.org/3.7/library/datetime.html#datetime.datetime.utcoffset) in the datetime lib. When it could have just used utcoffset directly. It is code for code's sake.

Reply:
Yeah, I don't know how easy it is to make those methods use a specific time zone such as UTC, which is why I asked.  In Java I'd have created a TimeZone object with "UTC" as the time zone name, and just used that. I'm guessing there's some equivalent for Python that I don't know yet.

Reply:
That is dead on in python as well. There is a `datetime.utcnow()` function that would act similarly.

Reply:
The builtin datetime lib in python is sufficient here -- I've removed the custom class that overrode datetime methods and replaced it with timezone.utc

Reply:
@llyon-cb Just unresolving until the changes come through.

Reply:
Changes are live, now using builtin lib